### DISCUSSION TITLE
Is Gatsby Serve production ready?

### DISCUSSION THREAD

**Discussion by ZeldOcarina** [gatsbyjs/gatsby#36478] [Help] [answered]
Hi!
Right now I am simply serving the "public" folder generated by the Gatsby build with NGINX.
The Google speed score is not high enough and I wonder if introducing SSR might help.
I read before that gatsby serve wasn't production-ready but now that paragraph seemed to have disappeared?
Can we now use that in production?
Thanks for the clarification.

PS: I found the paragraph where that notation was removed:

> Server-Side Rendering requires a running NodeJS server. You can put NodeJS running gatsby serve behind a content delivery network (CDN) like [Fastly](https://www.fastly.com/), however that also requires additional infrastructure (like monitoring, logging, and crash-recovery).
https://www.gatsbyjs.com/docs/reference/rendering-options/server-side-rendering/#using-in-production

**Comment by KyleAMathews** [comment, marked as answer]
We only support `gatsby serve` as a development aid. https://www.gatsbyjs.com/plugins/gatsby-plugin-fastify/ is a project which is intended for production use.
